Engine/Main/Globals.h

```diff
@@ -63,56 +63,22 @@ enum class Fonts
 // Paths -------
 # define APP_ICON_PATH "Resources/Textures/engine_icon.png"
 
-# define TEXTURES_PATH "Assets/Textures/"
-# define SHADERS_PATH "Resources/Shaders/shaders.json"
-
-# define HOUSE_MODEL_PATH "Assets/Models/baker_house/BakerHouse.fbx"
-# define DEFAULT_MATERIAL_PATH "Resources/Materials/default.olmat"
-
-# define VIDEO_BILLBOARD_TEXTURE_PATH "Library/Metadata/35/3575636683"
-# define LIGHT_BILLBOARD_TEXTURE_PATH "Library/Metadata/26/2664975493"
-
-# define PRIMITIVE_CUBE_PATH "Library/Metadata/41/4135282941"
-# define PRIMITIVE_CYLINDER_PATH "Library/Metadata/40/4057255088"
-# define PRIMITIVE_SPHERE_PATH "Library/Metadata/35/3553469253"
-# define PRIMITIVE_TORUS_PATH "Library/Metadata/22/2202705559"
-# define PRIMITIVE_QUAD_PATH "Library/Metadata/35/3548710651"
-
-# define MAIN_SCENE_PATH "Library/Main.scene"
-# define DEFAULT_SCENE_PATH "Resources/Scenes/default_scene.scene"
-# define TMP_SCENE_PATH "Assets/Scenes/tmp_scene.scene"
-# define SCENE_ROOT_PATH "Assets/Scenes/"
-# define GAME_INPUT_PATH "Resources/GameInputs/game_inputs.inputs"
-# define NAVMESH_PATH "Resources/NavMeshes/"
-
-# define SCRIPT_LIST_PATH "Resources/Scripts/scripts_list.scripts"
-
-#if DEBUG
-# define SCRIPTS_DLL_PATH "Resources/Scripts/GamePlaySystemDebug.dll"
-# define SCRIPT_DLL_FILE "GamePlaySystemDebu_.dll"
-#elif GAME
-# define SCRIPTS_DLL_PATH "Resources/Scripts/GameplaySystem.dll"
-# define SCRIPT_DLL_FILE "GameplaySystem.dll"
-#else
-# define SCRIPTS_DLL_PATH "Resources/Scripts/GamePlaySystemRelease.dll"
-# define SCRIPT_DLL_FILE "GamePlaySystemReleas_.dll"
-#endif
-
-# define SCRIPT_TEMPLATE_FILE_CPP "Resources/Scripts/TemplateScript.cpp"
-# define SCRIPT_TEMPLATE_FILE_H "Resources/Scripts/TemplateScript.h"
-# define SCRIPT_PATH "../GamePlaySystem/src/Script/"
-
- 
+# define VIDEO_BILLBOARD_TEXTURE_PATH "Resources/Textures/video-solid.png"
+# define LIGHT_BILLBOARD_TEXTURE_PATH "Resources/Textures/lightbulb-solid.png"
```

Comment:
this should be moved also to the path atlas right?

Reply:
Yes, you are right!

Reply:
(I'me deletign them!)